aiohomekit/controller/ip/connection.py

```diff
@@ -114,12 +112,19 @@ async def send_bytes(self, payload: bytes) -> HttpResponse:
         timeout_handle = loop.call_at(loop.time() + 30, self._handle_timeout, result)
         timeout_expired = False
         try:
+            self.transport.write(payload)
             return await result
-        except asyncio.TimeoutError:
-            timeout_expired = True
+        except (asyncio.TimeoutError, BaseException) as ex:
+            # If we get a timeout or any other exception then we need to
+            # close the connection as we are now out of sync with the device
+            # and any future requests will fail since the encryption counters
+            # will be out of sync.
```

Comment:
We might be able to redo the pair verify and start over but I doubt devices handle that well so I think the safest thing to do is close and reopen to ensure good state